main/exercice/upload_exercise.php

```diff
@@ -151,6 +151,7 @@ function lp_upload_quiz_action_handling() {
     $feedback_false_index = array();
     $number_questions = 0;
     $question_description_index = array();
+    $noNegativeScore_index = array();
```

Comment:
Variables MUST be in *all* $camelCaseWithoutDash now :-) (there's another one a bit below)